libs/core/src/lib/table/directives/table-status-indicator.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, Input } from '@angular/core';
+
+export type TableStatuses = 'valid' | 'warning' | 'information' | 'error'
+
+@Directive({
+    selector: '[fdTableStatusIndicator], [fd-table-status-indicator]',
+    host: {
+        class: 'fd-table__cell--status-indicator',
+        '[class.fd-table__cell--status-indicator--valid]': 'status === "valid"',
+        '[class.fd-table__cell--status-indicator--warning]': 'status === "warning"',
+        '[class.fd-table__cell--status-indicator--information]': 'status === "information"',
+        '[class.fd-table__cell--status-indicator--error]': 'status === "error"',
+    }
+})
+export class TableStatusIndicatorDirective {
```

Comment:
Is there an example showing usage of status indicator?

Reply:
Yes
![Screen Shot 2020-07-10 at 10 38 05 AM](https://user-images.githubusercontent.com/2471874/87177811-7b249000-c299-11ea-890c-fceb32fda5c8.png)